draft-ietf-jsonpath-base.md

```diff
@@ -402,7 +406,7 @@ elements from an array, giving a start position, an end position, and
 an optional step value that moves the position from the start to the
 end.
 
-Filter expressions `?<boolean expr>` select certain children of an object or array, as in:
+Filter expressions `?<expr>` select certain children of an object or array, as in:
```

Comment:
Why not use `logical-expr` instead and elsewhere?
```suggestion
Filter expressions `?<logical-expr>` select certain children of an object or array, as in:
```

Reply:
I liked the reduced weight, but went with the suggestion (throughout) in 58e7ed0